File: business/src/main/java/com/pij/lottieshow/interactor/UnknownFileException.java
package com.pij.lottieshow.interactor;

/**
 * <p>Created on 13/04/2017.</p>
 * @author Pierrejean
 */

class UnknownFileException extends RuntimeException {

    @SuppressWarnings("WeakerAccess")
    public UnknownFileException() {
        super();
    }

    @SuppressWarnings("WeakerAccess")
    public UnknownFileException(Throwable cause) {
        super(cause);
    }

    @SuppressWarnings("WeakerAccess")
    public UnknownFileException(String format) {
        super(format);
    }

    @SuppressWarnings("WeakerAccess")
    public UnknownFileException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: business/src/main/java/com/pij/lottieshow/interactor/UnknownFileException.kt
package com.pij.lottieshow.interactor

/**
 *
 * Created on 13/04/2017.
 * @author Pierrejean
 */

internal class UnknownFileException : RuntimeException {

    constructor() : super() {}

    constructor(cause: Throwable) : super(cause) {}

    constructor(format: String) : super(format) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}
}
